packages/openapi-generator/src/parser/document.ts

```diff
@@ -35,7 +35,9 @@ export async function parseOpenApiDocument(
   options: ParserOptions
 ): Promise<OpenApiDocument> {
   const clonedContent = JSON.parse(JSON.stringify(fileContent));
-  const document = (await parseBound(clonedContent)) as OpenAPIV3.Document;
+  const document = (await SwaggerParser.parse(
```

Comment:
[q] Why?

Reply:
parseBound was a workaround that is not necessary anymore